foo() {
    err2: int[][3]
}